### ISSUE TITLE
[Feature] BrowserContext uses the existing image js file

### ISSUE THREAD

**Issue by xiaowenhuman** [microsoft/playwright#6633]
BrowserContext uses the existing image js file.
I need a lot of BrowserContext. In order to save memory, how can I make BrowserContext use the cached BrowserContext?

**Comment by avodovnik** [comment]
As per [our documentation](https://playwright.dev/docs/core-concepts/#browser-contexts) browser contexts are fast and cheap to create, and in fact, can be used to parallelize isolated test executions.

I'm not entirely sure what you mean by "existing image js file"

**Comment by mxschmitt** [comment]
Closing for now since it seemed stale, if you still experience any issue feel free to reopen or create a new issue.

**Comment by VikramTiwari** [comment]
Hey @avodovnik, is there some documentation around best practices for running tests. Especially in parallel. I was thinking about reusing page but if contexts are cheap to create, it makes more sense to do just that.

Also, are there some numbers associated to these operations. I am looking for something like: it takes 1 second to launch a new context on a 1vCPU, 1GB RAM machine.

**Comment by mxschmitt** [comment]
Not yet since the test runner is in progress. Contexts are cheap to create (less than 50ms for chromium) and give you the benefit of having isolation between tests. This ensures they are not depending on each-other.

It's recommended to have a worker per CPU. And each browser takes around 256-512mb of memory from my experience but there are no official numbers yet.

**Comment by VikramTiwari** [comment]
These are very helpful numbers! Thanks @mxschmitt! Now that test runner is released, do you think there will be another resource for these?
